internal/common/voting/election.go

```diff
@@ -12,14 +15,41 @@ type Election struct {
 	votes         [][]shared.ClientID
 }
 
-type ElectionVotingMethod = int
+type ElectionVotingMethod int
```

Comment:
has the = been useless this whole time? or did this actually change something?

Reply:
`type A int` -> A is an integer type
`type B = int` -> B _is_ integer

Very subtle semantic difference--the former means that A is a new type that has the properties of an int. B on the other hand is equivalent to int.